### PR TITLE
[13.0][FIX] sale_stock_available_info_popup: right context

### PR DESCRIPTION
**Pull request by chienandalu**
We must ensure the right context is provided to the product in order to compute the right quantities in relation to the other values of the popup.

cc @Tecnativa TT35179

please review @victoralmau @pedrobaeza

### CODE REVIEW

**Comment by pedrobaeza** [comment]
You always forget to put the version in the PR title (not in the commit - there it shouldn't be put -).

**Review by pedrobaeza — APPROVED** [review]

**Comment by chienandalu** [comment]
> You always forget to put the version in the PR title (not in the commit - there it shouldn't be put -).

I know, sorry :face_with_spiral_eyes:

**Review by victoralmau — APPROVED** [review]

File: sale_stock_available_info_popup/models/sale_order.py [hunk 1]
```diff
@@ -18,9 +18,15 @@ def _compute_immediately_usable_qty_today(self):
         qty_processed_per_product = defaultdict(lambda: 0)
         self.immediately_usable_qty_today = False
         for line in self.sorted(key=lambda r: r.sequence):
+            if line.order_id.commitment_date:
+                date = line.order_id.commitment_date
```
Comment (rousseldenis) [inline sale_stock_available_info_popup/models/sale_order.py:22]: What is making the decision to take the commitment date and not the order date?

Shouldn't both computations in different fields ?

Comment (chienandalu) [reply sale_stock_available_info_popup/models/sale_order.py:22]: Hi, @rousseldenis It comes from core: https://github.com/odoo/odoo/blob/13.0/addons/sale_stock/models/sale_order.py#L243-L246

Comment (rousseldenis) [reply sale_stock_available_info_popup/models/sale_order.py:22]: Could you add a test with commitment_date ?

Comment (chienandalu) [reply sale_stock_available_info_popup/models/sale_order.py:22]: Added test for that condition and simplified test declaration using `Form`

**Comment by pedrobaeza** [comment]
Merging as the petition has been attended:

/ocabot merge patch

Can you fw-port it to 14?

**Pull request merged**